Code2Gether-Discord-Bot.Library/BusinessLogic/GitHubJoinTeamLogic.cs

```diff
@@ -42,11 +38,13 @@ public override async Task<Embed> ExecuteAsync()
             return embed;
         }
 
-        private async Task<EmbedContent> JoinGitHubTeam()
+        private async Task<HttpResponseMessage> PutNewMemberToTeamAsync()
         {
-            var githubClient = new GitHubClient(_githubAuthToken);
-            var response = await githubClient.AddOrUpdateTeamMembershipForUserAsync(_githubOrganizationName, _teamSlug, _username, Team.Role.member);
-            return ParseHttpResponseToEmbedContent(response);
+            const string json = "{\"role\":\"member\"}";
```

Comment:
Start using `Json.SerializeObject` to return this string?

Reply:
Sounds good